platform/darwin/src/MGLPointCollection.h

```diff
@@ -0,0 +1,49 @@
+#import <Foundation/Foundation.h>
+#import <CoreLocation/CoreLocation.h>
+
+#import "MGLOverlay.h"
+#import "MGLShape.h"
+
+/**
+ The `MGLPointCollection` class is used to define an array of coordinates.
```

Comment:
Emphasize that these coordinates are disconnected (as opposed to MGLMultiPoint).